tests/test_views_search.py

```diff
@@ -93,6 +95,50 @@ def test_pagination(app, indexed_records, search_url):
         assert 'prev' in data['links']
 
 
+def test_page_links(app, indexed_records, search_url):
+    """Test Link HTTP header on multi-page searches."""
+    with app.test_client() as client:
+        # Limit records
+        res = client.get(search_url, query_string=dict(size=1, page=1))
+        assert_hits_len(res, 1)
+
+        links = {
+            k: v for (k, v) in
+            map(lambda s: re.findall(r'<(.*)>; rel="(.*)"', s)[0][::-1],
+                [x for x in res.headers['Link'].split(', ')])
+        }
+
+        data = get_json(res)['links']
+        assert 'self' in data \
+               and 'self' in links \
+               and data['self'] == links['self']
+        assert 'next' in data \
+               and 'next' in links \
+               and data['next'] == links['next']
+        assert 'prev' not in data \
+               and 'prev' not in links
+
+        # Assert next URL before calling it
+        first_url = links['self']
+        next_url = links['next']
+        parsed_url = parse_url(next_url)
+        assert parsed_url['qs']['size'] == ['1']
+        assert parsed_url['qs']['page'] == ['2']
+
+        # Access next URL
+        res = client.get(to_relative_url(next_url))
+        assert_hits_len(res, 1)
+        links = {
+            k: v for (k, v) in
+            map(lambda s: re.findall(r'<(.*)>; rel="(.*)"', s)[0][::-1],
```

Comment:
This line is the same as L106-109 so can be put in it's own function (a la ``parse_link_header``).